### DISCUSSION TITLE
Export to XLS issue

### DISCUSSION THREAD

**Discussion by gallowayLm** [Power-Components/livewire-powergrid#1089] [Q&A]
Hello,

I have followed the instructions of the documentation regarding the upgrade from version 3 to version 4.

1. I have added to config/livewire-powergrid.php the 'exportable' => ['default' => 'openspout_v4']
2. I have completed the installation of openspout package
3. Finally I  am using the Trait WithExport in my Powergrid Table as shown below in the setup method

![Screenshot from 2023-08-09 15-48-34](https://github.com/Power-Components/livewire-powergrid/assets/109161140/82956fa5-e0a7-4cab-a048-35dde2886a5e)

When I am tying the action to export I receive the following error:
ERROR: Class "" not found {"userId":1,"exception":"[object] (Error(code: 0): Class \"\" not found at /app/vendor/power-components/livewire-powergrid/src/Traits/WithExport.php:229)

I have also tried composer install and composer dump-autoload after the openspout/openspout installation. Is anyone facing similar issues?

Thanks in advance

**Comment by gallowayLm** [comment]
Actually I had not complete the export array in the config/livewire-powergrid.php 
I was missing blue highlighted
![Screenshot from 2023-08-10 13-20-29](https://github.com/Power-Components/livewire-powergrid/assets/109161140/66db6494-d8f9-4f79-88e1-22f1ef221a8f)

Everything works smoothly.

**Reply by KonstantinosVazaios** [reply]
I have been trying so hard finding a solution... Thank you!
